Sources/Turf/BoundingBox.swift

```diff
@@ -9,8 +9,9 @@ public struct BoundingBox: Codable {
         guard coordinates?.count ?? 0 > 0 else {
             return nil
         }
+        let startValue = (minLat: coordinates!.first!.latitude, maxLat: coordinates!.first!.latitude, minLon: coordinates!.first!.longitude, maxLon: coordinates!.first!.longitude)
```

Comment:
Abbreviations should be avoided according to [Swift’s API design guidelines](https://swift.org/documentation/api-design-guidelines/) but min/max and lat/lon is a common standard and this implementation is private so it's fine.

Reply:
@frederoni : Agreed on the abbrevs. I generally follow the guideline that variable names are proportional to the size of the scope they're used in. I.e. small scope, all visible on screen means a variable name can be abbreviated if following common practice.